classes/class-wc-connect-shipping-label.php

```diff
@@ -52,14 +52,21 @@ protected function get_items_as_individual_packages( $order ) {
 			return $packages;
 		}
 
-		protected function get_packaging_data( WC_Order $order ) {
+		protected function get_packaging_metadata( WC_Order $order ) {
 			$shipping_methods = $order->get_shipping_methods();
 			$shipping_method = reset( $shipping_methods );
 			if ( ! $shipping_method || ! isset( $shipping_method[ 'wc_connect_packages' ] ) ) {
-				return $this->get_items_as_individual_packages( $order );
+				return FALSE;
```

Comment:
We've been using the lowercase version of true and false (WP coding standards don't address this directly, but all the examples use lowercase too at https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/ )